src/main/java/underdogs/devbie/recommendation/dto/RecommendationRequest.java

```diff
@@ -0,0 +1,15 @@
+package underdogs.devbie.recommendation.dto;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import underdogs.devbie.recommendation.domain.RecommendationType;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
```

Comment:
저희 컨벤션 정한대로 수정 부탁드려요~

```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Getter
```